docs/src/developing/lookup-tables.md

```diff
@@ -132,7 +130,7 @@ transactionV0.sign([payer]);
 
 // send and confirm the transaction
 // (NOTE: There is NOT an array of Signers here; see the note below...)
-const txid = await web3.sendAndConfirmTransaction(connection, transactionV0);
+const txidV0 = await web3.sendAndConfirmTransaction(connection, transactionV0);
```

Comment:
There is really no such thing as a "v0 signature". The value returned from the `sendAndConfirmTransaction` function is always just a [TransactionSignature](https://solana-labs.github.io/solana-web3.js/types/TransactionSignature.html) aka a string.

Reply:
Makes sense. I'll change the variable name in the console.log below instead to 'txid'